tests/fixtures/cognito_auth.py

```diff
@@ -1,5 +1,5 @@
 env_mock = {
     "COGNITO_AUTHENTICATION": "1",
-    "ALLOWED_PUBLIC_KEYS": '{"keys": [{"kid": "12345", "n": "xyz"}, {"kid": "54321", "n": "xyz"}]}',
+    "ALLOWED_PUBLIC_KEYS": '{"keys": [{"kid": "12345", "n": "xyz"}, {"kid": "54321", "n": "xyz"}]}',  # noqa: E501
```

Comment:
let's rather do

```
('{"keys": [{"kid": "12345", "n": "xyz"}'
'{"kid": "54321", "n": "xyz"}]}')
```

Reply:
black come to diffrent sollution but () helped.